projects/igniteui-angular/src/lib/progressbar/progressbar.component.ts

```diff
@@ -173,7 +172,7 @@ export abstract class BaseProgressDirective {
      * ```
      */
     public set valueInPercent(value: number) {
-        this._valueInPercent = value;
+        value = toValue(value, this._max);
```

Comment:
This setter does nothing. See if possible to remove it, or move it to the base class.

Reply:
One option is to delete it from BaseProgressDirective, since in the extending class (IgxLinearProgressBarComponent) we are calling the setter of the value.

Reply:
Oh, what I Show is actually in the base class. So, just move the `set valueInPercent` from extended to base class, using the implementattion from the base class.